### PR TITLE
Remove db

### PR DESCRIPTION
**Pull request by cawasser**
removed the database in favor of using edn config files. This supports more dynamic flexibility - can add/remove messages and types on the fly, and protect proprietary message formats since they don't need to be in the repo

### CODE REVIEW

File: src/clj/chocolate/db/core.clj [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
-
-(conman/bind-connection *db* "sql/queries.sql")
+;(defstate ^:dynamic *db*
+;          :start (conman/connect! {:jdbc-url (env :database-url)})
+;          :stop (conman/disconnect! *db*))
+;
+;(conman/bind-connection *db* "sql/queries.sql")
 
 
 (extend-protocol jdbc/IResultSetReadColumn
```
Comment (gpierce3) [inline src/clj/chocolate/db/core.clj:16]: Do we need this since we've removed the db?

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,8 @@
 pom.xml
 dev-config.edn
 test-config.edn
+local_protobuf.clj
+*-local.edn
 *.jar
 *.class
 /.lein-*
```
Comment (gpierce3) [inline .gitignore:12]: I know we aren't including the .protos in the repo.
I forget why we decided not to add *.proto to gitignore.

**Review by gpierce3 — APPROVED** [review]
All in all an elegant solution.
Keeps us focused on the messaging aspect w/o worrying about 'db' stuff.
Will also be easier to play with since we just have to update text instead of modifying a db.

File: src/clj/chocolate/db/core.clj (hunk 1 above)
Comment (cawasser) [reply src/clj/chocolate/db/core.clj:16]: I left it in under the assumption that we would one day start using the DB again for some reason.

File: .gitignore (hunk 2 above)
Comment (cawasser) [reply .gitignore:12]: we ADO include the `person.proto` and `message.proto` files so we can't blanket ignore *.proto files

**Review by austinpaine — APPROVED** [review]

**Pull request merged**